### ISSUE TITLE
my domain isn't working after installing the framework and updating hes config

### ISSUE THREAD

**Issue by gotgits** [susanBuck/e2-fall21#59]
My question is could someone share the command that will locate my /e2 directory and contents from SSH:hes-on-DigitalOcean that I see under sites available in VSCode.

The reason is I also ran into issues today adding the zipfoods framework, and in the process of updating the hes config file and installing zipfoods which I saw with all of its contents, I have broken my visibility to all of it, can't get back to my remote development folders at all now and I am at the point of a small meltdown. I am wondering how far back do I have to go if I cannot see even e2 directory anymore and will I break things even further accidentally creating a new ssh key pair.

I have never managed remote sites only worked in them. I have gone back to the subdomains sites-enabled and everything was fine but I lost all connection in the browser, so I kept saw everything after restarting nginx and restart with the new framework server array saw a list of contents in CL Terminal, everything was fine and then I just misplaced all of it.

edit: I can login to my namecheap dashboard and all is well. The Advanced DNS IP address matches digitalocean droplet.

I checked to see if anything was browser visible with nginx/sites-enabled sucessfully, I reviewed the contents list everything was great and even had the zipfoods subdomain show in the list before I moved or saved over the hes config file, my confusion came with duplicate html directories. I deleted and reinstalled the framework and thought I corrected that.

My DigitalOcean ip address is now not working either.
Could this just be a Sunday thing? It was all perfect this morning, I was seeing and testing all sorts of things in my /e2, and I don't cache between sessions.

So as previously stated, my question is could someone share the command that will locate my e2 directory and contents from SSH:hes-on-DigitalOcean that I see under sites available in VSCode? I just feel like it's all still there I am just not reaching it.

I would like to think it's not really a crisis, just a misstep that is compounding. In the meantime I am revisiting Week 2 notes and video. I have also tried several CL options that were fine, and now I cannot relocate my remote server.

-Rose

**Comment by susanBuck** [comment]
Hi @gotgits - To confirm, you are able to connect to the server, you're just trying to locate the e2 folder on the server?

If so, while connected to the server, you can run the following command in Terminal and it should open a new VSCode window that is a) connected to the server and b) showing your `e2` directory in the FIle Browser:

```
code /var/www/e2
```

If that's not working - or I'm not 100% understanding your question - just let me know and we'll go from there.

We can also connect on Zoom anytime tomorrow after 11am ET if that's at all helpful.

Regardless - try not to panic - we always have a backup of your code on Github, and the server is something that can always be recovered/straightened out.

**Comment by susanBuck** [comment]
Also, @gotgits - here are instructions you can follow to grant me access to your server: https://hesweb.dev/e2/notes/appendix/instructor-server-access

If I can connect to your server, I'll be better able to help your troubleshoot. 👍

**Comment by gotgits** [comment]
Yes now I can connect on vs code thank you.
However, I don’t understand how I messed up the hes file and why the URLs
aren’t working. At least now I can look at the files and check.

I was planning this week to revisit earlier notes/videos, I just didn’t
think it would be this far back as all of the infrastructure really felt
straightforward to me. I had downloaded my repository from GitHub as well.
I’m using this experience as time for review and some documenting.

**Comment by susanBuck** [comment]
I'd have to see your _hes_ file to understand better why it's not working.

You can open it with the following command:

```
code /etc/nginx/sites-available/hes
```

You can copy/paste the full contents in your reply here and I can take a look.

**Comment by gotgits** [comment]
Good morning,
I added your SSHkey in VSCode Terminal while in ***@***.***:/var/www/e2
There were no responses to that command so I am not clear from your
instructions on that page if I should be somewhere else. I will check the
original installation SSH key notes in the meantime.

I can take a zoom meeting at your convenience. Daytime is better for me to
minimize interruptions.

What I now see on my remote server that seems to be missing altogether is
the index.html and perhaps an index.php and I thought there was a folder
html as well, and that is where the trouble began when installing the
framework.

My current config hes file:

```
server {
    listen 80;
    server_name e2practice.metrognome.me;
    root /var/www/e2/practice/;

    index index.php index.html;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }ngin

    location ~ \.php$ {
        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
        include fastcgi_params;
    }
}
server {
    listen 80;
    server_name e2p1.metrognome.me;
    root /var/www/e2/p1/;

    index index.php index.html;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
        include fastcgi_params;
    }
}
server {
    listen 80;
    server_name e2p2.metrognome.me;
    root /var/www/e2/p2/;

    index index.php index.html;

    location ~ \.php$ {
        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
        include fastcgi_params;
    }
}
server {
    listen 80;
    server_name e2p3.metrognome.me;
    root /var/www/e2/p3/;

    index index.php index.html;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
        include fastcgi_params;
    }
}
server {
    listen 80;
    server_name e2rps.metrognome.me;
    root /var/www/e2/rps/;

    index index.php index.html;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
        include fastcgi_params;
    }
}
```

**Comment by susanBuck** [comment]
Hi @gotgits -

I was able to successfully log into your server. I'm going to check over things and will reply shortly with what I find out.

**Comment by susanBuck** [comment]
@gotgits - I'd like to connect on Zoom so we can troubleshoot this together. 

I have a meeting scheduled with another student at 10am ET, but I'm available now before then or any time after that (let's say 11am).

Let me know what works best for you.

My Zoom URL is https://harvard.zoom.us/my/susanbuck

**Comment by gotgits** [comment]
11:00 am sounds good. I will be there.

**Comment by gotgits** [comment]
I learned that the power the hes server config file has is great. The coincidence of an old domain name helpdesk issue resurfacing at the same time had unfortunate timing because it detracted from the fact that I had 4 letters ngin that snuck between brackets and while trying different commands I did not understand the subtle reference to line 12 in the response from the CL $ nginx -t which pointed this detail out. These stray letters caused my ip address to be disabled as well. In hindsight I may have noticed this if I had looked at the dif on github where it might have been highlighted.
